reesericci.md

```diff
@@ -0,0 +1,51 @@
+---
+name: "@reesericci & @ItIsTheLucas"
+project: "Foosball Scoreboard"
+---
+
+# Foosball Scoreboard
+
+## Summary
+
+We are going to build an IoT scoreboard for the foosball club at our high school that will automatically connect to the rating system we have. It will be basically a strip of NeoPixels for each team and then an arcade button you can hit when you score. We will also have a screen and interface for selecting who's playing and the max point value, to be able to communicate that to the rating system. A stretch goal would be to implement tournaments as well, so it automatically knows who's playing during a tournament.
+
+## Plan
+
+- I will start designing in CAD software the cases for all the different parts like the lights for the scoreboard, the button, screen, and rotary encoders. Then I will start printing them on my 3d printer with the filament. While I'm doing that, Lucas will start wiring things up like the neopixels and screen to the metro, and make a power board so it only needs one power supply.
```

Comment:
Can you please start assembly during the 10-day building period? If so, please move this to a seperate item (esigning 3D prints & printing is ok)

Reply:
They were going to be done concurrently during the 10 days

Reply:
I was doing the software CAD stuff and @ItIsTheLucas  assembling things.